### PR TITLE
Simplify page layout examples

### PR DESCRIPTION
**Pull request by taysea**
[Preview](https://deploy-preview-1250--keen-mayer-a86c8b.netlify.app/templates/page-layouts#sidebar-header-and-footer)
#### What does this PR do?
This is a proposed start to how we can simplify the page layouts examples to better then themselves to copy and paste by users. A small amount of added styling feels necessary to successfully portray page sections but this minimizes unnecessary gaps and background colors and adds comments directly on any lines of code that should be removed by the user.

#### Where should the reviewer start?
Any file.

#### How should this be manually tested?
Check the deploy of page layouts.

#### What are the relevant issues?
Related to https://github.com/grommet/hpe-design-system/issues/1240
Closes https://github.com/grommet/hpe-design-system/issues/1253

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible.

### CODE REVIEW

**Comment by halocline** [comment]
@taysea From my perspective I think this definitely streamlines the page layout examples, eliminating the extraneous colors, gaps, rounding. I would be fine with heading this direction.

@vavalos5, @rajasekar-ramasamy, @gregfuruiye, @ericsoderberghp, @MikeWalrath  - it would be good to get your eyes to confirm whether @taysea's proposed modifications meet your expectations.

**Comment by taysea** [comment]
Link to discussion can be found: https://hpe.slack.com/archives/GPPLUK6LR/p1604962122042800

**Comment by ericsoderberghp** [comment]
I wonder if the borders should be dashed instead of solid, to help clarify that the borders should not be used when replicating?

Also, are you thinking of changing the thumbnails at the top of the page as well?

**Comment by taysea** [comment]
> I wonder if the borders should be dashed instead of solid, to help clarify that the borders should not be used when replicating?
> 
> Also, are you thinking of changing the thumbnails at the top of the page as well?

I like the idea of dashed. Let me try that out. And I had initially thought of leaving the previews at the top of page because it seemed like on a smaller scale the existing format might provide a quicker-high-level interpretation but I am also happy to align the styling with the actual examples.

**Comment by vavalos5** [comment]
> I wonder if the borders should be dashed instead of solid, to help clarify that the borders should not be used when replicating?
> 
> Also, are you thinking of changing the thumbnails at the top of the page as well?

I also like the idea of having dashed lines. I'm wondering if we need to just create some quick explorations in Figma to view them side-by-side?

**Comment by taysea** [comment]
> > I wonder if the borders should be dashed instead of solid, to help clarify that the borders should not be used when replicating?
> > Also, are you thinking of changing the thumbnails at the top of the page as well?
> 
> I also like the idea of having dashed lines. I'm wondering if we need to just create some quick explorations in Figma to view them side-by-side?

It'd look something like this, I like the dashed because I think most people would know that's an indication of "placeholder" space.
Screen Shot 2020-11-10 at 8 49 42 AM

**Comment by ericsoderberghp** [comment]
> And I had initially thought of leaving the previews at the top of page because it seemed like on a smaller scale the existing format might provide a quicker-high-level interpretation but I am also happy to align the styling with the actual examples.

I'd recommend reaching out to Edgar to see what his thoughts are on this. My inclination is to align, just to try to avoid any confusion.

File: aries-site/src/examples/cardPreviews/page-layouts/HeaderFooterPreview.js [hunk 1]
```diff
@@ -3,17 +3,21 @@ import { Box, defaultProps } from 'grommet';
...
+    <Box fill>
       <Box
         height={defaultProps.theme.global.edgeSize.medium}
-        background="green!"
-        round="xsmall"
+        // remove in prod, for demo purposes only
+        border={{ color: 'border', style: 'dashed' }}
+      />
+      <Box
+        // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/HeaderFooterPreview.js:13]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/HeaderFooterPreview.js [hunk 2]
```diff
@@ -3,17 +3,21 @@ import { Box, defaultProps } from 'grommet';
 
 export const HeaderFooterPreview = () => {
   return (
-    <Box gap="xsmall" fill>
+    <Box fill>
       <Box
         height={defaultProps.theme.global.edgeSize.medium}
-        background="green!"
-        round="xsmall"
+        // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/HeaderFooterPreview.js:9]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/HeaderFooterPreview.js [hunk 3]
```diff
@@ -3,17 +3,21 @@ import { Box, defaultProps } from 'grommet';
...
+        border={{ color: 'border', style: 'dashed' }}
+        flex
       />
-      <Box background="orange" round="xsmall" flex />
       <Box
         height={defaultProps.theme.global.edgeSize.medium}
-        background={{ color: { light: 'blue', dark: 'blue!' } }}
-        round="xsmall"
+        // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/HeaderFooterPreview.js:19]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/HeaderOnlyPreview.js [hunk 4]
```diff
@@ -3,13 +3,17 @@ import { Box, defaultProps } from 'grommet';
 
 export const HeaderOnlyPreview = () => {
   return (
-    <Box fill gap="xsmall">
+    <Box fill>
       <Box
         height={defaultProps.theme.global.edgeSize.medium}
-        background="green!"
-        round="xsmall"
+        // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/HeaderOnlyPreview.js:9]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/HeaderOnlyPreview.js [hunk 5]
```diff
@@ -3,13 +3,17 @@ import { Box, defaultProps } from 'grommet';
...
+        border={{ color: 'border', style: 'dashed' }}
+      />
+      <Box
+        // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/HeaderOnlyPreview.js:13]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderFooterPreview.js [hunk 6]
```diff
@@ -3,23 +3,27 @@ import { Box, defaultProps } from 'grommet';
 
 export const SidebarHeaderFooterPreview = () => {
   return (
-    <Box fill direction="row" gap="xsmall">
+    <Box fill direction="row">
       <Box
         basis={defaultProps.theme.global.edgeSize.medium}
-        background={{ color: { light: 'purple', dark: 'purple!' }, dark: true }}
-        round="xsmall"
+        // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderFooterPreview.js:9]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderFooterPreview.js [hunk 7]
```diff
@@ -3,23 +3,27 @@ import { Box, defaultProps } from 'grommet';
...
+        border={{ color: 'border', style: 'dashed' }}
       />
-      <Box fill gap="xsmall">
+      <Box fill>
         <Box
           height={defaultProps.theme.global.edgeSize.medium}
-          background="green!"
-          round="xsmall"
+          // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderFooterPreview.js:15]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderFooterPreview.js [hunk 8]
```diff
@@ -3,23 +3,27 @@ import { Box, defaultProps } from 'grommet';
...
+          border={{ color: 'border', style: 'dashed' }}
+        />
+        <Box
+          // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderFooterPreview.js:19]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderFooterPreview.js [hunk 9]
```diff
@@ -3,23 +3,27 @@ import { Box, defaultProps } from 'grommet';
...
+          border={{ color: 'border', style: 'dashed' }}
+          flex
         />
-        <Box background="orange" round="xsmall" flex />
         <Box
           height={defaultProps.theme.global.edgeSize.medium}
-          background={{ color: { light: 'blue', dark: 'blue!' } }}
-          round="xsmall"
+          // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderFooterPreview.js:25]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderPreview.js [hunk 10]
```diff
@@ -3,19 +3,23 @@ import { Box, defaultProps } from 'grommet';
 
 export const SidebarHeaderPreview = () => {
   return (
-    <Box fill direction="row" gap="xsmall">
+    <Box fill direction="row">
       <Box
         basis={defaultProps.theme.global.edgeSize.medium}
-        background={{ color: { light: 'purple', dark: 'purple!' }, dark: true }}
-        round="xsmall"
+        // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderPreview.js:9]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderPreview.js [hunk 11]
```diff
@@ -3,19 +3,23 @@ import { Box, defaultProps } from 'grommet';
...
+        border={{ color: 'border', style: 'dashed' }}
       />
-      <Box fill gap="xsmall">
+      <Box fill>
         <Box
           height={defaultProps.theme.global.edgeSize.medium}
-          background="green!"
-          round="xsmall"
+          // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderPreview.js:15]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderPreview.js [hunk 12]
```diff
@@ -3,19 +3,23 @@ import { Box, defaultProps } from 'grommet';
...
+          border={{ color: 'border', style: 'dashed' }}
+        />
+        <Box
+          // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/SidebarHeaderPreview.js:19]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js [hunk 13]
```diff
@@ -3,28 +3,28 @@ import { Box, defaultProps } from 'grommet';
 
 export const StickyHeaderPreview = () => {
   return (
-    <Box fill direction="row" gap="xsmall">
+    <Box fill direction="row">
       <Box
         basis={defaultProps.theme.global.edgeSize.medium}
-        background={{ color: { light: 'purple', dark: 'purple!' }, dark: true }}
-        round="xsmall"
+        // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js:9]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js [hunk 14]
```diff
@@ -3,28 +3,28 @@ import { Box, defaultProps } from 'grommet';
...
+        border={{ color: 'border', style: 'dashed' }}
       />
       <Box fill>
         <Box
           height={defaultProps.theme.global.edgeSize.medium}
-          background="green!"
-          round={{ corner: 'top', size: 'xsmall' }}
+          // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js:15]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js [hunk 15]
```diff
@@ -3,28 +3,28 @@ import { Box, defaultProps } from 'grommet';
...
+          border={{ color: 'border', style: 'dashed' }}
         />
-        <Box gap="xsmall" fill>
+        <Box fill>
           <Box
-            background="orange"
-            round={{ corner: 'bottom', size: 'xsmall' }}
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js:20]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js [hunk 16]
```diff
@@ -3,28 +3,28 @@ import { Box, defaultProps } from 'grommet';
...
+            border={{ color: 'border', style: 'dashed' }}
             flex
           />
           <Box
             height={defaultProps.theme.global.edgeSize.medium}
-            background={{ color: { light: 'blue', dark: 'blue!' } }}
-            round="xsmall"
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/cardPreviews/page-layouts/StickyHeaderPreview.js:26]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js [hunk 17]
```diff
@@ -11,7 +11,8 @@ export const HeaderFooterExample = () => {
             fill="horizontal"
             pad="small"
             height="xxsmall" // remove in prod, for demo purposes only
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js:14]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js [hunk 18]
```diff
@@ -22,7 +23,8 @@ export const HeaderFooterExample = () => {
             flex={false}
             pad="small"
             height="xlarge" // remove in prod, for demo purposes only
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js:26]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js [hunk 19]
```diff
@@ -32,7 +34,8 @@ export const HeaderFooterExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js:37]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js [hunk 20]
```diff
@@ -10,7 +10,8 @@ export const HeaderOnlyExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js:13]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js [hunk 21]
```diff
@@ -21,7 +22,8 @@ export const HeaderOnlyExample = () => {
             fill={undefined}
             flex={false}
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js:25]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js [hunk 22]
```diff
@@ -11,7 +11,8 @@ export const SidebarHeaderExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js:14]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js [hunk 23]
```diff
@@ -22,7 +23,8 @@ export const SidebarHeaderExample = () => {
             fill={undefined}
             flex={false}
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js:26]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js [hunk 24]
```diff
@@ -46,7 +48,8 @@ const AppSidebar = () => {
       /* Min height is not needed in mobile contexts */
       height={size !== 'small' ? { min: '100%' } : undefined}
       pad="small"
-      border="border-weak" // remove in prod, for demo purposes only
+      // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js:51]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js [hunk 25]
```diff
@@ -19,7 +19,8 @@ export const SidebarHeaderFooterExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js:22]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js [hunk 26]
```diff
@@ -30,7 +31,8 @@ export const SidebarHeaderFooterExample = () => {
             fill={undefined}
             flex={false}
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js:34]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js [hunk 27]
```diff
@@ -41,7 +43,8 @@ export const SidebarHeaderFooterExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js:46]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js [hunk 28]
```diff
@@ -65,7 +68,8 @@ const AppSidebar = () => {
       /* Min height is not needed in mobile contexts */
       height={size !== 'small' ? { min: '100%' } : undefined}
       pad="small"
-      border="border-weak" // remove in prod, for demo purposes only
+      // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js:71]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 29]
```diff
@@ -18,7 +18,8 @@ export const StickyHeaderExample = () => {
           background="background-front"
           fill="horizontal"
           pad="small"
-          border="border-weak" // remove in prod, for demo purposes only
+          // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:21]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 30]
```diff
@@ -30,7 +31,8 @@ export const StickyHeaderExample = () => {
             fill={undefined}
             flex={false}
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:34]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 31]
```diff
@@ -41,7 +43,8 @@ export const StickyHeaderExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:46]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 32]
```diff
@@ -65,7 +68,8 @@ const AppSidebar = () => {
       /* Min height is not needed in mobile contexts */
       height={size !== 'small' ? { min: '100%' } : undefined}
       pad="small"
-      border="border-weak" // remove in prod, for demo purposes only
+      // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:71]:
```suggestion
        // remove dashed border in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 33]
```diff
@@ -41,7 +43,8 @@ export const StickyHeaderExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
+            border={{ color: 'border', style: 'dashed' }}
             height="xxsmall" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:48]:
```suggestion
            height="xxsmall" // remove height="xxsmall" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js [hunk 34]
```diff
@@ -21,7 +22,8 @@ export const HeaderOnlyExample = () => {
...
+            border={{ color: 'border', style: 'dashed' }}
             height="xlarge" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js:27]:
```suggestion
            height="xlarge" // remove height="xlarge" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js [hunk 35]
```diff
@@ -11,7 +11,8 @@ export const SidebarHeaderExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
+            border={{ color: 'border', style: 'dashed' }}
             height="xxsmall" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js:16]:
```suggestion
            height="xxsmall" // remove height="xxsmall" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js [hunk 36]
```diff
@@ -22,7 +23,8 @@ export const SidebarHeaderExample = () => {
             fill={undefined}
             flex={false}
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
+            border={{ color: 'border', style: 'dashed' }}
             height="xlarge" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderExample.js:28]:
```suggestion
            height="xlarge" // remove height="xlarge" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js [hunk 37]
```diff
@@ -19,7 +19,8 @@ export const SidebarHeaderFooterExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
+            border={{ color: 'border', style: 'dashed' }}
             height="xxsmall" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js:24]:
```suggestion
            height="xxsmall" // remove height="xxsmall" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js [hunk 38]
```diff
@@ -30,7 +31,8 @@ export const SidebarHeaderFooterExample = () => {
             fill={undefined}
             flex={false}
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
+            border={{ color: 'border', style: 'dashed' }}
             height="xlarge" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js:36]:
```suggestion
            height="xlarge" // remove height="xlarge" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js [hunk 39]
```diff
@@ -41,7 +43,8 @@ export const SidebarHeaderFooterExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
+            border={{ color: 'border', style: 'dashed' }}
             height="xxsmall" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/SidebarHeaderFooterExample.js:48]:
```suggestion
            height="xxsmall" // remove height="xxsmall" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 40]
```diff
@@ -18,7 +18,8 @@ export const StickyHeaderExample = () => {
           background="background-front"
           fill="horizontal"
           pad="small"
-          border="border-weak" // remove in prod, for demo purposes only
+          // remove in prod, for demo purposes only
+          border={{ color: 'border', style: 'dashed' }}
           height="xxsmall" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:23]:
```suggestion
          height="xxsmall" // remove height="xxsmall" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js [hunk 41]
```diff
@@ -30,7 +31,8 @@ export const StickyHeaderExample = () => {
             fill={undefined}
             flex={false}
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
+            border={{ color: 'border', style: 'dashed' }}
             height="xlarge" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js:36]:
```suggestion
            height="xlarge" // remove height="xlarge" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js [hunk 42]
```diff
@@ -10,7 +10,8 @@ export const HeaderOnlyExample = () => {
             background="background-front"
             fill="horizontal"
             pad="small"
-            border="border-weak" // remove in prod, for demo purposes only
+            // remove in prod, for demo purposes only
+            border={{ color: 'border', style: 'dashed' }}
             height="xxsmall" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderOnlyExample.js:15]:
```suggestion
            height="xxsmall" // remove height="xxsmall" in prod, for demo purposes only
```

File: aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js [hunk 43]
```diff
@@ -32,7 +34,8 @@ export const HeaderFooterExample = () => {
...
+            border={{ color: 'border', style: 'dashed' }}
             height="xxsmall" // remove in prod, for demo purposes only
```
Comment (halocline) [inline aries-site/src/examples/templates/page-layouts/HeaderFooterExample.js:39]:
```suggestion
            height="xxsmall" // remove height="xxsmall" in prod, for demo purposes only
```

**Review by halocline — COMMENTED** [review]
Made suggestions for comments to be a little bit more specific. When the "remove" comment precedes the line to be removed, it can be unclear whether I am to remove just the next line or do I also remove the [(e.g. flex)] which also follows.

Minor, but could help. Open to leaving if you feel the comments are clear as is.

**Pull request merged**